modules/survey_accounts/php/surveyaccountsprovisioner.class.inc

```diff
@@ -0,0 +1,75 @@
+<?php declare(strict_types=1);
+/**
```

Comment:
You can delete these file comments now if you want as long as we merge #6046 first.